package/yast2-registration.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Oct 13 15:30:58 UTC 2015 - ancor@suse.com
+
+- Using suse_register in AutoYaST profile doesn't longer produce
```

Comment:
NP: just a better wording: `Using "suse_register" section in AutoYaST profile does not produce an error message anymore`